infra/templates/osdu-r3-resources/README.md

```diff
@@ -77,76 +77,68 @@ git commit --allow-empty -m "Initializing the Flux Manifest Repository"
 
 Generate the [deploy key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) using `ssh-keygen`. The public portion of the key pair will be uploaded to GitHub as a deploy key.
 
-Run: `ssh-keygen -b 4096 -t rsa -f ~/.ssh/gitops-ssh-key`.
-
 ```bash
-$ ssh-keygen -b 4096 -t rsa -f ~/.ssh/gitops-ssh-key
-Generating public/private rsa key pair.
-Enter passphrase (empty for no passphrase):
-Enter same passphrase again:
-Your identification has been saved in /Users/jmspring/.ssh/gitops-ssh-key.
-Your public key has been saved in /Users/jmspring/.ssh/gitops-ssh-key.pub.
-The key fingerprint is:
-SHA256:jago9v63j05u9WoiNExnPM2KAWBk1eTHT2AmhIWPIXM jmspring@kudzu.local
-The key's randomart image is:
-+---[RSA 4096]----+
-|.=o.B= +         |
-|oo E..= .        |
-|  + =..oo.       |
-|   . +.*o=       |
-|    o * S..      |
-|   . * . .       |
-|... o ... .      |
-|...  .o+.. .     |
-|  .o..===o.      |
-+----[SHA256]-----+
-kudzu:azure-simple jmspring$
+AZURE_VAULT=<common_vault_name>
+KEY_NAME=gitops-ssh-key
+
+# Generate gitops-ssh-key
+ssh-keygen -b 4096 -t rsa -f $KEY_NAME
+
+# Save gitops-ssh-key
+az keyvault secret set --vault-name $AZURE_VAULT -n "${KEY_NAME}" -f "${KEY_NAME}"
+az keyvault secret set --vault-name $AZURE_VAULT -n "${KEY_NAME}-pub" -f "${KEY_NAME}.pub"
+
+# Show Public gitops-ssh-key
```

Comment:
Will this show the key in our pipeline logs?

Reply:
No because this is something that exists in the readme that is done manually.  The pipeline requires ssh files that are then used as secure files in Azure Devops.